tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -0,0 +1,187 @@
+"""Test class for Capsule Loadbalancer
+
+:Requirement: Loadbalancer capsule
+
+:CaseAutomation: Automated
+
+:CaseLevel: Integration
+
+:CaseComponent: Capsule
+
+:Assignee: akjha
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import DataFile
+from robottelo.utils.installer import InstallerCommand
+
+pytestmark = [pytest.mark.no_containers, pytest.mark.destructive]
+
+
+@pytest.fixture(scope='module')
+def get_hosts_from_broker(rhel7_contenthost_module, module_lb_capsule):
```

Comment:
I think you should directly use them in fixtures/tests, you dont need this additional fixture / orchestration.

Reply:
Done.